### PR TITLE
Alexmnet news

### PR DESCRIPTION
**Pull request by AlexMNet**

### CODE REVIEW

File: themes/ra-theme/assets/css/header.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@ header {
     aspect-ratio: 1;
     border: 0;
     background-image: url("/images/icon-hamburger.svg");
-    background-color: transparent;
+    background-color: 'transparent';
```
Comment (mckoss) [inline themes/ra-theme/assets/css/header.scss]: no quotes

File: themes/ra-theme/assets/css/header.scss [hunk 2]
```diff
@@ -53,7 +53,7 @@ header {
         position:fixed;
         z-index: 90;
         inset: 0 0 0 30%;
-        background-color: hsla(0, 0%, 100%, 0.5);
+        background-color: 'hsla(0, 0%, 100%, 0.5)';
```
Comment (mckoss) [inline themes/ra-theme/assets/css/header.scss]: no quotes

File: themes/ra-theme/assets/css/summary.scss [hunk 3]
```diff
@@ -3,13 +3,17 @@ $summary-break: 600px;
 ul.summary {
   margin: 1rem auto;
   padding: 0;
-  max-width: 800px;
+  max-width: 990px;
```
Comment (mckoss) [inline themes/ra-theme/assets/css/summary.scss]: Please add as a breakpoint in the breakpoint.scss file.

**Review by mckoss — CHANGES_REQUESTED** [review]
A few changes to make.

**Comment by AlexMNet** [comment]
- Removed the quotes from the header sass file
- Removed center alignment on images
- Added max-summary-width of 990px to breakpoints

**Review by mckoss — APPROVED** [review]
Looks great, Alex.  "LGTM"

**Pull request merged**